src/main/groovy/org/shipkit/internal/gradle/e2e/E2ETestingPlugin.java

```diff
@@ -23,10 +26,16 @@
  */
 public class E2ETestingPlugin implements Plugin<Project> {
 
+    public static final String RUN_ALL_E2E_TESTS_TASK = "runAllE2ETests";
+
     public void apply(final Project project) {
-        E2ETest e2eTest = project.getExtensions().create("e2eTest", E2ETest.class, project);
-        // TODO hardcoded for now
-        e2eTest.create("https://github.com/mockito/shipkit-example");
+        TaskMaker.task(project, RUN_ALL_E2E_TESTS_TASK, new Action<Task>() {
```

Comment:
To simplify the public API how about we merge the functionality of e2eTest extension and this new 'runAllE2ETests' task. Instead of those 2, we could just create "e2eTest" task and configure it in the following way:

```Gradle
//build.gradle
e2eTest {
   repository(“https://github.com/mockito/shipkit-example“)
   repository(“https://github.com/mockito/mockito“)

   //or:
   repository(“https://github.com/mockito/mockito“, “https://github.com/mockito/shipkit-example“)
}
```